wren-ui/src/components/diagram/index.tsx

```diff
@@ -109,6 +109,20 @@ const ReactFlowDiagram = forwardRef(function ReactFlowDiagram(
     setForceRender(!forceRender);
   };
 
+  const dispatchMouseEvent = (event: React.MouseEvent | MouseEvent) => {
+    if (event.isTrusted) {
+      const mouseDownEvent = new MouseEvent('mousedown', {
```

Comment:
The dropdown menu does not work if we dispatch the `mousedown` event before the event flow reaches the dropdown element.

https://github.com/user-attachments/assets/6650ee27-a6e9-4d68-91c1-071b399d6f4e

In this case, we need to add a condition if the user clicks inside the dropdown component.

For example,
```tsx
const dropdowns = document.querySelectorAll('.ant-dropdown');
```